stdlib/public/core/Availability.swift

```diff
@@ -45,3 +45,39 @@ public func _stdlib_isOSVersionAtLeast(
   return false._value
 #endif
 }
+
+
+/// Returns the version number for the Swift Standard Library that the code
+/// calling this was originally compiled with.
+///
+/// The version number is an arbitrary `Double` value that is monotonically
+/// increasing between toolchain releases. It does not correlate with toolchain
+/// or OS version numbers, and it is not a semantic version number.
```

Comment:
If it's arbitrary and opaque and has no real correlation, why a Double?

Reply:
It's not really opaque; it's intended as a way to create two-level version number.

The fractional part leaves some room to fork the version number progression when we open a new release branch.

```
master:                   3727.0, 3728.0, 3729.0, ...
swift-5.1-branch:         3728.0, 3728.1, 3728.2, ..., 3728.9, 3728.91, ...    
```

Of course, a pair of integers would do this a lot cleaner. I'll change it.